mods/ts/rules/civilian-infantry.yaml

```diff
@@ -270,13 +272,16 @@ VISC_LRG:
 		HP: 500
 	Armament:
 		Weapon: SlimeAttack
+		FireDelay: 10
```

Comment:
Neither these nor `doggie`s will attack me when placed as creeps in the map editor.  What's going on?